### DISCUSSION TITLE
Could you tell me how to use fixedwing-airsim?

### DISCUSSION THREAD

**Discussion by youthfrog** [AOS55/Fixedwing-Airsim#2] [Q&A] [answered]
With great enthusiasm to your discussion [https://github.com/microsoft/AirSim/issues/2955](url), I come to your github. I really admire your work.

Recently I am doing a psychological experiment on the human-computer interaction of fixed-wing drones. I want to use a model of a fixed-wing aircraft. I read the entire discussion mentioned carefully, and I don't quite understand the current progress of your fixed-wing drone. At the same time I want to use your model, but I'm not clear how to use it. Can you give me a brief introduction?

I'd be appreciate a lot for your help.

**Comment by AOS55** [comment, marked as answer]
Hi YouthFrog, sure Ill work on some proper documentation later on today. But to get started the following should work:
- Install airsim from the microsoft repo (the vanilla airsim is fine) 
- Install JSBSim from the JSBSim repo (Ill put the x8 file up later on today). 
- I would start with the blocks environment as the package is already there. 
- You then run the programme in CV mode `{
  "SettingsVersion": 1.2,
  "SimMode": "ComputerVision"
}
`
- Then in fixedwing-airsim just run main to start with this will allow you to use the AP and path planning class. You can then change the profile around to get it to fly where you want it to go. 

Sorry for the lack of docmentation Ill get something up today along with a more detailed status report.
